http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/Boot.scala

```diff
@@ -252,11 +265,11 @@ object Boot extends IOApp with LazyLogging {
                                         dataprocConfig.dataprocDefaultRegion,
                                         dataprocConfig.dataprocZone)
 
-      googlePublisher <- GooglePublisher.resource[F, LeoPubsubMessage](publisherConfig)
+      googlePublisher <- GooglePublisher.resource[F](publisherConfig)
 
       publisherQueue <- Resource.liftF(InspectableQueue.bounded[F, LeoPubsubMessage](pubsubConfig.queueSize))
 
-      publisherStream = publisherQueue.dequeue through googlePublisher.publish
+      publisherStream = Stream.eval(Logger[F].info(s"Initializing publisher for ${publisherConfig.projectTopicName}")) ++ (publisherQueue.dequeue through convertToPubsubMessagePipe through googlePublisher.publishNative)
```

Comment:
What is `publishNative`? Will look at wb-libs PR..

Reply:
maybe it can have a better name. The idea is its a pipe processes on raw `PubsubMessage` (a good lib data type), this allows users more freedom however they want to customize the message, such as adding attributes etc.